swap/src/protocol/alice/event_loop.rs

```diff
@@ -148,25 +147,25 @@ where
                 swarm_event = self.swarm.next_event() => {
                     match swarm_event {
                         SwarmEvent::Behaviour(OutEvent::SpotPriceRequested { request, channel, peer }) => {
-                            if self.resume_only {
```

Comment:
I'd handle the resume-only case here in the loop instead of further down in call-chain.

Reply:
The resume-only case only concerns the spot price protocol and not the complete eventloop.
Now that we have a dedicated `NetworkBehaviour` for Alice the event loop is not even aware of the resume-only mode anymore :)
Note that we can rename the flag to "don't accept swaps` or something similar if you prefer - so the name fits better to the context of the spot-price protocol.